### DISCUSSION TITLE
Mosaico email images not showing up on Apple Mail

### DISCUSSION THREAD

**Discussion by viccenteno** [voidlabs/mosaico#711] [Q&A] [answered]
I have been dumbfounded on why I only get the "question mark in a blue box" placeholder for my images in Apple Mail when I use Mosaico. If I open it in say, Gmail, it is fine, but not in Apple Mail. Also, if I use the traditional CiviMail message template, I don't get this problem.

**Comment by bago** [comment]
Does it happen to you when using mosaico.io demo or when using a custom installation of the mosaico library?

**Comment by viccenteno** [comment]
It happens when I use a custom installation. I think part of the problem is with our web host. They disabled ModSecurity and it allowed us to embed images (through code on a text block) from our website. However if I use the image block of any of the existing templates, the images don't show up on Apple Mail. It is a bit cumbersome to code the images in, but I will take it. I am not sure where the exact issue lies (our web host keeps saying the problem is not with them), but there is a work around now.

**Comment by bago** [comment, marked as answer]
Well, this sounds like an issue with your hosting and nothing about mosaico: i doubt we can help.

Apple does "image proxy preloading" so they do request the images linked in emails using a CDN and then if you open the email they get the image from their own CDN: it is called "mail privacy protection". Of course if your host blocks their CDN, then images in Mac/iPhone won't work.
